app/javascript/styles/mastodon/components.scss

```diff
@@ -4662,6 +4661,10 @@ a.status-card.compact:hover {
 
 .search {
   position: relative;
+
+  @media screen and (min-width: 631px) {
```

Comment:
I am not sure it needs to be conditional at all, I fail to see when it is used when below 632px in width.

Reply:
It looks different when it's 631px and when it's 630px. Anything under 630px will be displayed in mobile mode. In this case, the search area will look completely different. So I explicitly made it only apply if it's 631px or greater.

Reply:
Yes, I mean I can't see the `search` class being used at all below the 630px breakpoint, so I don't think the CSS needs to apply conditionally!